integration/container_stats_test.go

```diff
@@ -74,6 +75,64 @@ func TestContainerStats(t *testing.T) {
 	testStats(t, s, containerConfig)
 }
 
+// Test to verify if the consumed stats are correct.
+func TestContainerConsumedStats(t *testing.T) {
+	t.Logf("Create a pod config and run sandbox container")
+	sbConfig := PodSandboxConfig("sandbox1", "stats")
+	sb, err := runtimeService.RunPodSandbox(sbConfig, *runtimeHandler)
+	require.NoError(t, err)
+	defer func() {
+		assert.NoError(t, runtimeService.StopPodSandbox(sb))
+		assert.NoError(t, runtimeService.RemovePodSandbox(sb))
+	}()
+
+	testImage := GetImage(ResourceConsumer)
+	img, err := imageService.PullImage(&runtime.ImageSpec{Image: testImage}, nil, sbConfig)
+	require.NoError(t, err)
+	defer func() {
+		assert.NoError(t, imageService.RemoveImage(&runtime.ImageSpec{Image: img}))
+	}()
+
+	// consume 1 GB memory for 60 seconds.
```

Comment:
would 100MB suffice?

Reply:
Sure, but bear in mind that a container by itself will also consume memory. From what I saw, on Windows it consumes ~13 MB, which is negligible when compared to 1 GB.

For 100 MB, we'd have to make the check more precise.